README.md

```diff
@@ -20,7 +20,7 @@ $ miqversions
 | Fine           | 5.8.z                        | 4.5        | 2.3.x | 9.5.x      |
 | Gaprindashvili | 5.9.z                        | 4.6        | 2.3.x | 9.5.x      |
 | Hammer         | 5.10.z                       | 4.7        | 2.4.x | 9.5.x      |
-| Ivanchuck      | 5.11.z                       | 4.8        | 2.4.x | 9.5.x      |
```

Comment:
???  It's spell wrong.  Why change the PG version?  What am I missing?

Reply:
my brain isn't working... I thought this was already fixed... see https://github.com/chrisarcand/miqversions/pull/14 (no "chuck" typo and pg version is 10)

Reply:
yeah, I guess everything here was copy-pasta and kinda borked... fixed